Snakefile

```diff
@@ -219,6 +219,19 @@ rule cluster_network:
     # group: 'build_pypsa_networks'
     script: "scripts/cluster_network.py"
 
+
+rule add_extra_stores:
```

Comment:
```suggestion
rule add_additional_components:
```

```suggestion
rule add_extra_components:
```
I was thinking to make/name this rule more general to add any energy system components that should/cannot be added before clustering. E.g. another extension that would fit well in this rule is to build expansion options for a generic hydrogen network.